src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -4,36 +4,122 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants.MotorIDConstants;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Constants.CtreCanID;
+import frc.robot.Constants.RevCanID;
+import utilities.HowdyPID;
+import utilities.TOFSensorSimple;
 
 public class IntakeSubsystem extends SubsystemBase {
   /** Creates a new IntakeSubsystem. */
   private SparkMax intakeMotor;
 
+  private TalonFX pivotMotor;
+  private SparkMax conveyorMotor;
+  private HowdyPID pivotPID;
+  private Angle pivotSetpoint = kPivotRetractAngle;
+
+  private TOFSensorSimple sensor;
+
   public IntakeSubsystem() {
-    intakeMotor = new SparkMax(MotorIDConstants.kIntakeMotor, MotorType.kBrushed);
+    intakeMotor = new SparkMax(RevCanID.kIntakeMotor, MotorType.kBrushless);
+    pivotMotor = new TalonFX(CtreCanID.kPivotMotor);
+    conveyorMotor = new SparkMax(RevCanID.kConveyorMotor, MotorType.kBrushless);
+    pivotPID = new HowdyPID(kPivotP, kPivotI, kPivotD);
+    sensor = new TOFSensorSimple(RevCanID.kConveyorSensor, Inches.of(1));
+  }
+
+  private void setPivotMotor(double speed) {
+    pivotMotor.set(speed);
+  }
+
+  private void setConveyerMotor(double speed) {
+    conveyorMotor.set(speed);
+  }
+
+  private void setIntakeMotor(double speed) {
+    intakeMotor.set(speed);
+  }
+
+  public Angle getPivotPositionCommand() {
+    return pivotMotor.getPosition().getValue();
+  }
+
+  public Command extendPivotCommand() {
+    return runOnce(() -> pivotSetpoint = kPivotExtendAngle);
+  }
+
+  public Command retractPivotCommand() {
+    return runOnce(() -> pivotSetpoint = kPivotRetractAngle);
   }
 
   // Made a command to spin clockwise
-  public Command IntakeCommand() {
-    return startEnd(() -> intakeMotor.set(kSpeed), () -> intakeMotor.set(0));
+  public Command intakeCommand() {
+    return startEnd(() -> setIntakeMotor(kIntakeSpeed), () -> setIntakeMotor(0));
   }
 
   // Made a command to spin counter clockwise
-  public Command OuttakeCommand() {
-    return startEnd(() -> intakeMotor.set(-kSpeed), () -> intakeMotor.set(0));
+  public Command outtakeCommand() {
+    return startEnd(() -> setIntakeMotor(-kIntakeSpeed), () -> setIntakeMotor(0));
+  }
+
+  public Command pivotDownCommand() {
+    return startEnd(() -> setPivotMotor(-kPivotSpeed), () -> setPivotMotor(0));
+  }
+  
+  public Command pivotUpCommand() {
+    return startEnd(() -> setPivotMotor(kPivotSpeed), () -> setPivotMotor(0));
+  }
+
+  /**
+   * Pushes game piece from conveyor into birdhouse
+   */
+  public Command conveyorFeed() {
+    return startEnd(() -> setConveyerMotor(-kConveyorSpeed), null);
+  }
+
+  public Command conveyorEject() {
+    return startEnd(() -> setConveyerMotor(kConveyorSpeed), () -> setConveyerMotor(0));
+  }
+
+
+  public Command intakeToBirdhouse() {
+    return pivotDownCommand()
+            .alongWith(intakeCommand())
+            .alongWith(conveyorFeed())
```

Comment:
I mentioned in the meeting, but I think we will need to change the `alongWith` Command compositions. This is because a subsystem can only do one Command at a time. `alongWith` only works with Commands of different Subsystems. 

Instead you will need to make a Command (either inline here or as another Command factory) that controls all three motors.